packages/astral/src/ValueObject/AttributeKey.php

```diff
@@ -6,7 +6,6 @@
 
 /**
  * @api
- * @deprecated use a custom AST Visitor instead, see https://phpstan.org/blog/preprocessing-ast-for-custom-rules
```

Comment:
I guess this deprecation should stay, as this ValueObject is no longer usefull?

Reply:
This value object is ok. Only parent/next/previous constants are deprecated. 
Those can be removed now as well :+1:

Reply:
ok - done.